pkgs/top-level/all-packages.nix

```diff
@@ -4568,7 +4568,7 @@ in
 
   isync = callPackage ../tools/networking/isync { };
 
-  itm-tools = callPackage ../development/tools/misc/itm-tools { };  
+  itm-tools = callPackage ../development/tools/misc/itm-tools { };
```

Comment:
Unrelated changes shouldn't be in this commit.

Reply:
Thank you! My Vim configuration deletes trailing whitespace upon file write. Now fixed.